pkg/proc/variables.go

```diff
@@ -1210,16 +1210,16 @@ func loadValues(vars []*Variable, cfg LoadConfig) {
 }
 
 // Extracts the value of the variable at the given address.
-func (v *Variable) loadValue(cfg LoadConfig) {
+func (v *Variable) LoadValue(cfg LoadConfig) {
```

Comment:
Please fixup doc comment now that this is exported (e.g. `LoadValue extracts ...`).

Reply:
no longer exporting this